Python/specialize.c

```diff
@@ -707,6 +707,10 @@ _Py_Specialize_LoadAttr(PyObject *owner, _Py_CODEUNIT *instr, PyObject *name, Sp
             return -1;
         }
     }
+    if (!_PyType_HasFeature(type, Py_TPFLAGS_VALID_VERSION_TAG)) {
```

Comment:
@markshannon, I wrapped the code in a macro, so in the future when you remove `Py_TPFLAGS_VALID_VERSION_TAG ` or decide otherwise, you can easily change just the macro to check for `tp_version == 0` instead.